tools/WixBuild.vcxproj.props

```diff
@@ -52,6 +53,8 @@
   <PropertyGroup>
     <ResourceOnlyDll Condition=" '$(ResourceOnlyDll)'=='' ">false</ResourceOnlyDll>
     <GenerateManifest Condition="$(ResourceOnlyDll)">false</GenerateManifest>
+    <WindowsSDKDesktopARMSupport Condition=" '$(Platform)'=='arm' and $(VS2017Available) ">true</WindowsSDKDesktopARMSupport>
```

Comment:
Is there an easy way to check whether ARM support was installed? Better to skip than fail the build, imo.